starter-core/src/main/java/io/micronaut/starter/options/JdkDistribution.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.options;
+
+public enum JdkDistribution {
+    ADOPT("adopt", "Adopt OpenJDK Hotspot"),
```

Comment:
where did you optain this list from? From github? Can you add a javadoc with a link?

Reply:
I got that list from SDKMAN on my M1 Mac. I just did a list `sdk list java` but it only gives me the ones for Apple Silicon. If there is an official list of distributions somewhere I'm not aware of it. However, I just found a list at the [SDKMAN website](https://sdkman.io/jdks), so I wonder which of these we do or don't want to enumerate (or all)?

Reply:
Update to reflect the distributions supported by github actions `setup-java`

see: https://github.com/actions/setup-java#supported-distributions

add javadoc `@see` with the above link to this enum.